docs/fides/docs/language/taxonomy/data_subjects.md

```diff
@@ -4,13 +4,13 @@ A Data Subject is a label that describes a segment of  individuals whose data yo
 
 ## Object Structure
 
-**fides_key**<span class="required"/>_string_
+**fides_key**<span class="required"/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_constrained string_
 
-A string token of your own invention that uniquely identifies this Data Subject. It's your responsibility to ensure that the value is unique across all of your Data Subject objects. The value should only contain alphanumeric characters and underbars (`[A-Za-z0-9_]`). 
+A string token of your own invention that uniquely identifies this Data Subject. It's your responsibility to ensure that the value is unique across all of your Data Subject objects. The value should only contain alphanumeric characters and underbars (`[A-Za-z0-9_.-]`).
```

Comment:
```suggestion
A custom string token that uniquely identifies this Data Subject. It must be unique across all Data Subject objects. The value may only contain alphanumeric characters, `_`, `.`, and `-` (it will be evaluated against the regular expression: `[A-z0-9_.-]`).
```